agent/containers/images/image-test-script.sh

```diff
@@ -312,7 +332,8 @@ wait_keypress 120
 
 if [ "$(podman ps | grep pbench-viz)" ]; then
     read -p "Would you like the live-metric-visualizer to be terminated? (y/[n]): " term_vis
-    if [[ -n $term_vis && $term_vis == "y" ]]
+    term_vis=$(echo $term_vis | tr '[:upper:]' '[:lower:]')
+    if [[ "$term_vis" == "y" || "$term_vis" == "yes" ]]
```

Comment:
This looks ripe for refactoring.  These three lines (and the other y/n prompts) could be just an `if` statement invoking a function:
```
    if yes_no_is_yes("Would you like the live-metric-visualizer to be terminated?", "n")
```
which invokes something like:
```
function yes_no_is_yes() {
    prompt=$1
    default=$2
    y_n=" (y/n)"
    read -p "${prompt} ${y_n/${default}/[${default}]}"
    lc_reply=$(tr '[:upper:]' '[:lower:]' <<< ${REPLY:-${default}})
    return [[ "${lc_reply}" == "y" || "${ls_reply}" == "yes" ]]
}
```

By the way, you might want to look into some of the other switches that `read` takes.  For instance, if you used the `-n 1` switch, which limits the reply to one character, then you could simplify the subsequent comparison (by just comparing it to `y`, or to `y` or `Y` and skip the translation to lower case).  And, for the prompts for things like filenames, you should consider using `-e`, which would enable command line editing for the user.